tests/Persisters/Serializers/NativeTest.php

```diff
@@ -42,19 +46,45 @@ public function build() : void
         $this->assertInstanceOf(Native::class, $this->serializer);
         $this->assertInstanceOf(Serializer::class, $this->serializer);
     }
-    
+
     /**
      * @test
      */
     public function serializeUnserialize() : void
     {
         $data = $this->serializer->serialize($this->persistable);
-        
+
         $this->assertIsString($data);
 
         $persistable = $this->serializer->unserialize($data);
 
         $this->assertInstanceOf(DummyClassifier::class, $persistable);
         $this->assertInstanceOf(Persistable::class, $persistable);
     }
+
+	/**
+	 * @return array<mixed>
+	 */
```

Comment:
lol you made the change before i could comment ... this works, but should the annotation technically be `array[]`?

btw `$ composer analyze` to run static analysis